src/components/CourseSearch.js

```diff
@@ -48,13 +75,42 @@ function CourseSearch() {
               <XchangeTable
                 headers={[
                   "UW Course Name",
-                  "Host School Course Name",
-                  "University",
-                  "Terms Offered",
+                  "Host University Course",
+                  "Host University",
+                  "Year Taken",
+                  "Program of Student"
                 ]}
-                colWidths={["25%", "30%", "25%", "20%"]}
-                tableBody={tempTableBody}
-                // TO-DO Connect to the course endpoint
+                colWidths={["25%", "20%", "25%", "10%", "20%"]}
```

Comment:
I think here you would want to pass the `numRows` similar to how it's done for `UniversitySearch.js`, so that we don't have "NaN" and "undefined" for pagination.

Reply:
Just did it 😄